### ISSUE TITLE
[Proposal] Ability to disable "No message selected" sign

### ISSUE THREAD

**Issue by KG-insky** [bkaankose/Wino-Mail#415]
We should be able to switch off "No message selected" sign especially since we can add custom wallpapers into the app that look awful when combined with this sign
![Снимок экрана 2024-10-12 002235](https://github.com/user-attachments/assets/b1a5c4bc-2189-45fb-82bc-fe7584a978e7)

**Comment by larrykvit** [comment]
I agree! Its quite obvious that a message isn't selected. And it totally wrecks feel of the background image.

**Comment by bkaankose** [comment]
This is fixed on the Store version I suppose. That panel is not visible on custom theme selections.
